config/common/culture/innovations.cwt

```diff
@@ -0,0 +1,86 @@
+types = {
+    type[innovation] = {
+        path = "game/common/culture/innovations"
+    }
+}
+
+innovation = {
+    ### Key to the cultural era this innovation belongs.
+    culture_era = <era>
+    ### Innovation group. Can be: culture_group_military, culture_group_civic or culture_group_regional.
+    group = enum[innovation_group]
+    ## cardinality = 0..1
+    ### Path to the icon to show. If not set will use the default icon defined in NGameIcons::DEFAULT_CULTURE_INNOVATION_TYPE_ICON_PATH
+    icon = filepath[gfx/interface/icons/culture_innovations]
+
+    ## cardinality = 0..1
+    ### Region where this innovation can start getting base progress. Empty means anywhere.
+    region = <region>
+    ## cardinality = 0..1
+    ### Trigger to check if it can be unlocked by the culture. In contrast to can_progress it will be hidden otherwise. Scope: culture Default: always = yes
+    potential = single_alias_right[trigger_clause]
+    ## cardinality = 0..1
+    ### To check if it can start being exposed. Scope: culture Default: always = yes
+    can_progress = single_alias_right[trigger_clause]
+
+    ## cardinality = 0..inf
+    ### Key of a building that can be unlocked. There can be more than one. This is only to show on the tooltip, it has to be maulually blocked on the object itself.
+    unlock_building = <building>
+    ## cardinality = 0..inf
+    ### Key of a decicion that can be unlocked. There can be more than one. This is only to show on the tooltip, it has to be maulually blocked on the object itself.
+    unlock_decision = <decision>
+    ## cardinality = 0..inf
+    ### Key of a casus belli that can be unlocked. There can be more than one. This is only to show on the tooltip, it has to be maulually blocked on the object itself.
+    unlock_casus_belli = <casus_belli>
+    ## cardinality = 0..inf
+    ### Key of a regiment that can be unlocked. There can be more than one. Actually does unlock the MaA.
+    unlock_maa = <regiment_type>
+    ## cardinality = 0..inf
+    ### Key of a law that can be unlocked. There can be more than one. This is only to show on the tooltip, it has to be maulually blocked on the object itself.
+    unlock_law = <law>
```

Comment:
These _might_ be effects and thus should go in the effects alias, but I don't see them documented in effects.log. Might be worth checking or filing an issue to check to see if these work in other effect contexts.

Reply:
They are not in the effects listed by the script_doc command. Also, most of these are just tooltips in reality.